src/Aspire.Dashboard/Components/Pages/Metrics.razor

```diff
@@ -10,7 +10,12 @@
 @inject IStringLocalizer<Dashboard.Resources.Metrics> Loc
 @inject IStringLocalizer<ControlsStrings> ControlsStringsLoc
 
-<PageTitle><ApplicationName ResourceName="@nameof(Dashboard.Resources.Metrics.MetricsPageTitle)" Loc="@Loc" /></PageTitle>
+<PageTitle>
+    <ApplicationName
+        AdditionalText="@PageViewModel.SelectedApplication.Id?.ReplicaSetName"
```

Comment:
If there is also an instrumented selected then the additonal text can combine both

e.g. just app: `frontend`, instrumented selected: `frontend - http.client.request.duration`